product-mini/platforms/zephyr/simple/CMakeLists.txt

```diff
@@ -31,10 +31,8 @@ if (NOT DEFINED WAMR_BUILD_LIBC_BUILTIN)
   set (WAMR_BUILD_LIBC_BUILTIN 1)
 endif ()
 
-if (NOT DEFINED WAMR_BUILD_LIBC_WASI)
-  # Disable libc wasi support by default
-  set (WAMR_BUILD_LIBC_WASI 0)
-endif ()
+# Disable libc wasi support on Zephyr
+set (WAMR_BUILD_LIBC_WASI 0)
```

Comment:
The orginal lines already disable libc-wasi by default, must we use hardcode here? Note that zephyr file/socket support is under development in branch `dev/zephyr_file_socket`.

Reply:
The libc-wasi synchronization primitive will also need some work, but I think for now maybe keeping the original config and adding more comments is preferred against hardcode.